src/app/tips/[id]/page.tsx

```
@@ -1,22 +1,25 @@
-import { ArticleJsonLd } from "next-seo";
-import React from "react";
-import Balancer from "react-wrap-balancer";
-import { POST_TYPE } from "src/types/enums";
 import { generatePostMetadata } from "@/common/metadata";
 import { getTipBySlug } from "@/common/tips";
 import { MdxRemote } from "@/components/Mdx";
 import NewsLetterForm from "@/components/NewsLetter";
 import NextNPrevious from "@/components/NextNPrevious";
 import TagList from "@/components/Tags";
 import { config } from "@/src/common/appconfig";
+import type { PageParams } from "@/src/types/types";
```

ESLint

[error] Expected "@/src/types/types" (type-internal) to come before "@/src/common/appconfig" (value-internal).

(perfectionist/sort-imports)